examples/typescript-plugin/README.md

```diff
@@ -0,0 +1,62 @@
+# Typescript Plugin Example
+
+This is an example plugin written in Typescript that serves as a base for more complex projects.
+
+## Requirements
+
+- A Strapi application running in V4 or above
+- Node
+
+## Get Started
+
+### Building the Plugin
+
+In order to use the plugin, you'll need to transpile the code to Javascript.
+
+To do so, you can use one of the following commands:
+
+1. `npm run build`
```

Comment:
To stay consistent with the rest of Strapi you should change this to  `yarn` instead of using `npm`.